lerobot/scripts/train.py

```diff
@@ -170,6 +171,7 @@ def log_train_info(logger: Logger, info, step, cfg, dataset, is_offline):
         f"lr:{lr:0.1e}",
         # in seconds
         f"updt_s:{update_s:.3f}",
+        f"dataloading_s:{dataloading_s:.3f}",
```

Comment:
```suggestion
        f"data_s:{dataloading_s:.3f}",  # if not ~0, you are bottlenecked by cpu or io
```

Reply:
@amandip7 could you apply this suggestion? and let's merge :)

Reply:
Done @Cadene Thanks!